magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/services/LambdaDiscovery.java

```diff
@@ -58,7 +59,7 @@ public void discover(ObjectMapper mapper, Session session, Region region, Emitte
         data.resourceId = function.revisionId();
         data.resourceName = function.functionName();
         data.resourceType = "AWS::Lambda::Function";
-        data.updatedIso = function.lastModified();
+//        data.updatedIso = Instant.parse(function.lastModified());
```

Comment:
Is the comment here intentional?

Reply:
Yes, function.lastModified() is in format that doesn't allows simple parsing